examples/move/bridge/sources/treasury.move

```diff
@@ -14,44 +14,44 @@ module bridge::treasury {
     use bridge::usdt::USDT;
     use sui::coin::TreasuryCap;
 
-    friend bridge::bridge;
+    /* friend bridge::bridge; */
 
     const EUnsupportedTokenType: u64 = 0;
     // const ENotSystemAddress: u64 = 1;
 
-    struct BridgeTreasury has store {
+    public struct BridgeTreasury has store {
         treasuries: ObjectBag
     }
 
-    public(friend) fun add_treasury_cap<T>(self: &mut BridgeTreasury, treasury_cap: TreasuryCap<T>) {
-        let type = type_name::get<T>();
-        object_bag::add(&mut self.treasuries, type, treasury_cap)
+    public(package) fun add_treasury_cap<T>(self: &mut BridgeTreasury, treasury_cap: TreasuryCap<T>) {
+        let `type` = type_name::get<T>();
+        object_bag::add(&mut self.treasuries, `type`, treasury_cap)
     }
 
-    public(friend) fun create(ctx: &mut TxContext): BridgeTreasury {
+    public(package) fun create(ctx: &mut TxContext): BridgeTreasury {
         // assert!(tx_context::sender(ctx) == @0x0, ENotSystemAddress);
         BridgeTreasury {
             treasuries: object_bag::new(ctx)
         }
     }
 
-    public(friend) fun burn<T>(self: &mut BridgeTreasury, token: Coin<T>, ctx: &TxContext) {
+    public(package) fun burn<T>(self: &mut BridgeTreasury, token: Coin<T>, ctx: &TxContext) {
     // public(friend) fun burn<T>(self: &mut BridgeTreasury, token: Coin<T>, ctx: &mut TxContext) {
```

Comment:
Not related to this PR I think, but somewhat confused by the commented-out function signature below this one? Might be best to remove unless there's a good reason to keep it?

Reply:
I don't see a good reason to have it, removing